[ASM] - Expander - Lowercase GridField Keys (#28873)
demisto/content: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert_README.md

Reviewer: Alphabetical order for list please.

Author: This was generated with SDK, want to put in a enhancement to have them set lists in alphabetical order?

| BypassDevCheck | Determine whether to bypass the Dev Check in automated remediation criteria: https://docs-cortex.paloaltonetworks.com/r/Cortex-XPANSE/Cortex-Xpanse-Expander-User-Guide/Automated-Remediation-Capabilities-Matrix<br/><br/>Set to "True" if you want to bypass.  Default is "False". | False | Optional |
| AcceptedRiskDs | Comma separated list of instance/VM IDs that are considered accepted risk and that should be closed. |  | Optional |
| AcceptedRiskProjects | Comma separated list of projects numbers that are considered accepted risk and that should be closed.  For example, the could be list of GCP projects and AWS accounts. |  | Optional |

> | AcceptedRiskOther | Comma separated list of other items that are considered accepted risk and that should be closed. For example, the could be list of folders numbers in GCP and subscription IDs in Azure. |  | Optional |

Reviewer 2: ```suggestion
| AcceptedRiskOther | Comma-separated list of other items that are considered an accepted risk and that should be closed. For example, a list of folders numbers in GCP and subscription IDs in Azure. |  | Optional |
```
